debian/kolibri.scripts-common

```diff
@@ -303,4 +310,5 @@ kolibri_preinst()
 kolibri_prerm()
 {
     stop_kolibri_service
+    clean_kolibri_pyc
```

Comment:
You could normalize this by replacing this line with just `py3clean -p kolibri` :)